snuba_sdk/mql/mql.py

```diff
@@ -107,7 +111,7 @@
 }
 
 
-def parse_mql(mql: str) -> MetricsQuery:
+def parse_mql(mql: str) -> Timeseries | Formula:
```

Comment:
the docstring is now out of date. I'm also not sure how or where MetricsQuery is constructed now. if not here

Reply:
The MetricsQuery would be constructed by whoever calls this function. The reason I removed this is because wrapping the parsed object in a MetricsQuery was largely useless. The MetricsQuery has nothing populated besides the `query` field, and is usually discarded anyways.

For example: I use this function in the visitor when validating that the MQL string is correct. That would mean calling `parse_mql` inside the `MetricsQuery` visitor,  and then discarding the `MetricsQuery` anyways.